core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -534,6 +534,17 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.CLIENT)
           .build();
+  public static final PropertyKey ZOOKEEPER_LEADER_CONNECTION_ERROR_POLICY =
+      new Builder(Name.ZOOKEEPER_LEADER_CONNECTION_ERROR_POLICY)
+          .setDefaultValue("STANDARD")
```

Comment:
Is there a downside to `SESSION`? Perhaps we should make that default?

Reply:
Only down-side is that it's not tested enough.

Reply:
Switched to SESSION as default.